dash/webapp/pages/toggles.js

```diff
@@ -57,6 +57,13 @@ const abTests = [
     description: '',
     range: [20, 30],
   },
+  {
```

Comment:
This would normally go into the A/B tests of the toggler app.
Although thinking, maybe it's worth having them as different types, and just denoting that here?

It's useful to have to the two concepts of a/b tests and feature toggles.

Reply:
As I understand it, we add it to both, in order that you can opt out of an A/B test condition if you wish?

Reply:
Nope, feature toggles are for things we want on, but can turn off at any stage as it's broken etc. We can do this (somewhat manually through S3) wiithout deploy. These are served to 100%. It can also be things like Google Analytics etc that you might want to turn off. Always on or off.

A/B tests are that, split traffic, time slotted bits of test functionality that should have some sort of measurement behind them.

Reply:
hmmm – is this lies then?

![image](https://user-images.githubusercontent.com/1394592/63173945-db369600-c038-11e9-805e-a6fe49c9404e.png)